src/tpm2_wrap.c

```diff
@@ -4666,7 +4917,7 @@ int wolfTPM2_SetCryptoDevCb(WOLFTPM2_DEV* dev, CryptoDevCallbackFunc cb,
         devId = rc;
         tpmCtx->dev = dev;
 
-        rc = wc_CryptoDev_RegisterDevice(devId, cb, tpmCtx);
+        rc = wc_CryptoCb_RegisterDevice(devId, cb, tpmCtx);
```

Comment:
is this required due to wolfCrypt recent library change?

Reply:
That is the old name that is mapped via macro in wolfCrypt. I am switching to the new name.